### DISCUSSION TITLE
Terminal en mi máquina

### DISCUSSION THREAD

**Discussion by DanteCely** [nodo-eafit-frontend/frontend-workshop-2024-1#7] [Polls]
Responde la encuesta para conocer si tenemos la terminal lista.

**Comment by vicoso27** [comment]
Profe yo descargue WSL 2 pero me presenta un error al ejecutarla, no se que hice mal, pero también. Tengo Git Bash y me trabaja bien.
No se si con el Git Bash pueda seguir trabajando o le sigo intentando al WSL2

**Reply by DanteCely** [reply]
GitBash es una alternativa si WSL2 no te sirve. Pero igual podemos mirar que está pasando. Comparteme porfa:
- Build de tu sistema operativo:  [Cómo saber la versión o build de Windows 10 que tengo
Cómo saber la versión o build de Windows 10 que tengo](https://www.adslzone.net/2017/05/10/como-saber-la-version-o-build-de-windows-10-que-tengo/)
- comparalo con los [prerequisitos ](https://learn.microsoft.com/es-es/windows/wsl/install) para instalar WSL o WSL2:
    - Para ejecutar los siguientes comandos, debe ejecutar Windows 10 versión 2004 y posteriores (**compilación 19041** y posteriores) o Windows 11. 
    
 - Iremos viendo que pasos nos faltaría

**Reply by vicoso27** [reply]
Profe con esa explicacion ya lo pude instalar, es mas instale 3 distribuciones y no se con cual quedarme o como borrar alguna.
Pero de resto ya funciona bien.

![image](https://github.com/nodo-eafit-frontend/frontend-workshop-2024-f04/assets/106107164/8924ed01-f226-4f3c-b311-bd0532559e5f)

**Reply by DanteCely** [reply]
@vicoso27 Yo uso la de Ubuntu, aunque las otras distribuciones funcionarían sin problema. La que elijas está bien, entiendo que con la de Debian las instalaciones son un poco diferentes pero nada que con documentación se pueda solucionar.

**Comment by vns31407** [comment]
Profe ya me descargo pero no se si quedo bien o le falta algo 
![Uploading IMG_20240529_222300.jpg…]()
![IMG_20240529_222252](https://github.com/nodo-eafit-frontend/frontend-workshop-2024-f04/assets/170677306/fa2d7f28-a61c-4b50-bdbe-6c6d1b62898e)

**Reply by DanteCely** [reply]
@vns31407 Ya estás dentro dell terminal WSL 🎊

**Comment by EAlvarez90** [comment]
Hola! profe una pregunta es que mi terminal el día de hoy no deja escribir y haciendo el proceso en visual studio que ayer nos explicaste no me aparece para seleccionarlo. 
![Captura de pantalla 2024-05-30 105900](https://github.com/nodo-eafit-frontend/frontend-workshop-2024-f04/assets/170678278/33f35bdc-41ef-447b-98e1-54bc7017a3e6)

**Reply by DanteCely** [reply]
Me preocupa que no esté corriendo el terminal. Entonces el terminal no te muestra ningún feedback para excribir, ¿correcto?

Trata de reinstalarlo con estos pasos:
[Install Ubuntu on WSL2](https://canonical-ubuntu-wsl.readthedocs-hosted.com/en/latest/guides/install-ubuntu-wsl2/)

Así que:
- Primero aseguremonos de que la terminal Ubuntu nos funcione.
- Luego miramos cómo el VS COde puede contener nuestro terminal instalado

**Reply by EAlvarez90** [reply]
Profe! 
lo volví a instalar y ya puedo escribir, pero aún no me aparece en VS Code 🫣 o se puede llamar diferente?

![Captura de pantalla 2024-05-30 155339](https://github.com/nodo-eafit-frontend/frontend-workshop-2024-f04/assets/170678278/784b2b67-1d22-4ad6-9203-010661a12728)

**Reply by DanteCely** [reply]
@EAlvarez90 Aquí veo que ya tienes el terminal disponible. Es el mismo `bash`, aunque escribe el comando `pwd` para saber en que directorio estás ubicada 🤔
